backend/src/test/java/ca/gov/dtsstn/cdcp/api/data/repository/ConfirmationCodeRepositoryIT.java

```diff
@@ -0,0 +1,39 @@
+package ca.gov.dtsstn.cdcp.api.data.repository;
+import static org.assertj.core.api.Assertions.assertThat;
+import java.time.Instant;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase.Replace;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.ActiveProfiles;
+import ca.gov.dtsstn.cdcp.api.config.DataSourceConfig;
+
+
+@DataJpaTest
+@ActiveProfiles("test")
+@Import({ DataSourceConfig.class })
+@AutoConfigureTestDatabase(replace = Replace.NONE)
+public class ConfirmationCodeRepositoryIT {
```

Comment:
Something that I don't like about this test is that it relies on data that is in the `data-init.sql` file.

I know that other tests do the same thing.. I feel that they're also not good tests.

What you should do, instead.. is do a `findById(..)` and verify that the data doesn't exist, then insert using `save(..)` and do another `findById(..)`

Reply:
...same idea with the `delete` method